background.js

```diff
@@ -1,6 +1,6 @@
 function setBadgeInfo(active = true) {
     chrome.action.setBadgeText({ text: !!active ? 'on' : 'off' })
-    chrome.action.setBadgeBackgroundColor({ color: !!active ? 'green': 'gray' }) 
+    chrome.action.setBadgeBackgroundColor({ color: !!active ? '#F5F5F5': '#F5F5F5' }) 
```

Comment:
@geraldopmj: Hey! 👋 Hmm... the background might be too light. It seems to almost blend into the extension bar. A user should immediately be able to determine the active state of the extension by a quick glance. A colourful badge immediately helps in this regard.

<img width="73" alt="image" src="https://github.com/cdrani/yqm/assets/18746599/e3f6fa48-88a4-43f1-89a1-c299f1927908">
<img width="73" alt="image" src="https://github.com/cdrani/yqm/assets/18746599/a287e341-fb01-485e-a4b9-accf8437bb1f">


I think the better approach here would be to follow the same path as the chorus extension action but adapt it use youtube colour instead of spotify's. On `on` state it would be white text on the youtube red background, then on `off` state, white text on darker grey background.


https://github.com/cdrani/chorus/blob/9f50b405b71a077f0b82918d2b5cb92623a420f9/background.js#L1-L5

Reply:
Hey! Ok, will do that!